app/scripts/contenteditor/ce-config.js

```diff
@@ -18,7 +18,25 @@ org.ekstep.contenteditor.config = {
 	corePluginsPackaged: true,
 	dispatcher: 'local',
 	localDispatcherEndpoint: '/app/telemetry',
-	previewURL: '/content/preview/preview.html'
+	previewURL: '/content/preview/preview.html',
+	cloudStorage: {
```

Comment:
How this config can be changed by any adopter?

Reply:
@vinukumar-vs We are not changing the way of passing the configuration to the editor. Just added new property to the existing configuration is still remain the same as is it.

Reply:
**For example:** 
```
window.context = {
  ....
  presigned_headers: {
      'x-ms-blob-type': 'BlockBlob'
  }
  ....         
}
```